runtime/cudaq/platform/default/rest/helpers/quantinuum/quantinuum.yml

```diff
@@ -20,6 +20,7 @@ config:
   # Define the JIT lowering pipeline
   jit-high-level-pipeline: "expand-measurements"
   jit-mid-level-pipeline: "quantinuum-gate-set-mapping"
+  jit-low-level-pipeline: "func.func(qubit-reset-before-reuse)"
```

Comment:
Is this meant to be a Quantinuum only pass?

Reply:
Yea, currently, it's the one that has this requirement.